bundles/org.openhab.voice.rustpotterks/src/main/java/org/openhab/voice/rustpotterks/internal/RustpotterKSService.java

```diff
@@ -62,27 +66,41 @@
         + " Keyword Spotter", description_uri = SERVICE_CATEGORY + ":" + SERVICE_ID)
 public class RustpotterKSService implements KSService {
     private static final String RUSTPOTTER_FOLDER = Path.of(OpenHAB.getUserDataFolder(), "rustpotter").toString();
+    private static final String RUSTPOTTER_RECORDS_FOLDER = Path.of(RUSTPOTTER_FOLDER, "records").toString();
     private final Logger logger = LoggerFactory.getLogger(RustpotterKSService.class);
-    private final ScheduledExecutorService executor = ThreadPoolManager.getScheduledPool("OH-voice-rustpotterks");
+    private final ExecutorService executor = ThreadPoolManager.getPool("voice-rustpotterks");
```

Comment:
Wil the threads still get an `OH-` prefix?

Reply:
That seems to be the case, it was displayed as "OH-OH-voice-rustpotterks" in the output of the `threads` console command.